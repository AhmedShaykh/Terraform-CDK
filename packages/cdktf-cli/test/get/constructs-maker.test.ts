

describe("constructsMaker", () => {
    describe("determineGoModuleName", () => {
        it("can determine ", async () => {
            expect(1).toMatchSnapshot();
        });
    });
});
