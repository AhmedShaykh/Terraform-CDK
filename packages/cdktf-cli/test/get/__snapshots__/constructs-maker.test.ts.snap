// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constructsMaker determineGoModuleName can determine  1`] = `1`;
